app/src/main/res/values/strings.xml

```diff
@@ -534,11 +534,6 @@
         <item quantity="one">You and 1 other person were added to the conversation</item>
         <item quantity="other">You and %1$d people were added to the conversation</item>
     </plurals>
-    <plurals name="last_message_other_added_self_user_and_others">
-        <item quantity="zero">You were added to the conversation</item>
-        <item quantity="one">You and 1 other person were added to the conversation</item>
-        <item quantity="other">You and %1$d people were added to the conversation</item>
-    </plurals>
```

Comment:
Just to double-check.

Are these removals intentional?

Reply:
yes, I was just lazy to create another separated PR to remove them